SignalServiceKit/protobuf/OWSSignalServiceProtos.proto

```diff
@@ -142,19 +142,25 @@ message SyncMessage {
     optional Type type = 1;
   }
 
-  message Read {
+  message ReadLinkedDevices {
```

Comment:
Renaming the `Read` message to `ReadLinkedDevices` to disambiguate from `ReadSender`.